config/config.yml.example

```diff
@@ -1,107 +1,211 @@
 development: &default
-  secret_token: somethingsecret
+  secret_token: 'somethingsecret'
+  app_name: 'Check'
   authorization_header: 'X-Check-Token'
-  twitter_consumer_key: test
-  twitter_consumer_secret: test
-  facebook_app_id: 123456
-  facebook_app_secret: test
-  slack_app_id: 123456
-  slack_app_secret: test
-  pender_url: http://localhost:3005
-  pender_url_private: http://localhost:3005
-  pender_key: test
-  elasticsearch_host: 127.0.0.1
-  elasticsearch_port: 9200
-  elasticsearch_log: false
-  elasticsearch_sync: false
+  checkdesk_base_url: 'http://localhost:3000'
+  checkdesk_base_url_private: 'http://localhost:3000'
   checkdesk_client: 'http://localhost:3333'
-  default_mail: root@localhost
-  checkdesk_base_url: http://localhost:3000
-  checkdesk_base_url_private: http://api:3000
-  send_welcome_email_on_registration: false
+  app_url: 'http://localhost:3333'
+  pender_url: 'http://localhost:3200'
+  pender_url_private: 'http://pender:3200'
+  pender_key: 'dev'
+  alegre_host: 'http://alegre:3100'
+  alegre_token: 'dev'
+  elasticsearch_host: 'elasticsearch'
+  elasticsearch_port: 9200
   elasticsearch_index:
-  # smtp mail settings
-  smtp_host: smtp.gmail.com # (google's default)
-  smtp_port: 587 # (google's default)
-  smtp_user: user@host.com # usually your gmail account
-  smtp_pass: 123456 # usually your gmail account password
-  pusher_app_id:
-  pusher_key:
-  pusher_secret:
-  pusher_cluster:
-  video_file_max_size:
-  uploaded_file_max_size:
-  image_min_width:
-  image_max_width:
-  image_min_height:
-  image_max_height:
+  elasticsearch_log: true
+  elasticsearch_sync: false
+  elasticsearch_cloud_id: 'qa:ZXUtd2VzdC0xLmF3cy5mb3VuZC5pbyQxZDkxYjk4YzZmZjQ0YjA0OGZhNDc2NzI4ZTQ2ZTAzMSRhMjE1ODRmZGVkOTM0MzFiYjkyOWU5OGQ3OWYyMzUxMQ=='
+  elasticsearch_user: 'elastic'
+  elasticsearch_password: 'HzzENVbibvxsl4LaykQqqOlW'
```

Comment:
This is the QA password, I don't think we need a password for the local instance, but please confirm with Aaron